README.md

```diff
@@ -64,30 +64,51 @@ Runs the `bin/create-version.js` script, which is used to create a new versioned
 
 **Compliance Testing**
 
-`yarn test:compliance https://api.example.com dist/latest/openapi.yaml`
+`yarn test:compliance`
 
-Executes the `bin/test-compliance.js` script to run compliance tests against the OpenAPI specification. This ensures that the specification is valid and conforms to the OpenAPI standard. It uses [`wiretap`](https://github.com/pb33f/wiretap) that acts like a proxy server to intercept requests and validate them against the OpenAPI spec.
+Runs a comprehensive compliance test suite against a live Cloud Foundry API. This script uses `wiretap` to proxy requests to the API, validating them against the OpenAPI specification in real-time. It clones the `capi-bara-tests` repository and executes its test suite against the proxied API.
+
+> :warning: Unfortunally wiretap seems to be quite instable it cannot properly handle multiform-data requests and sometimes just crashes with memory erros. Its currently as good as it is and helped a lot making the spec compliant.
```

Comment:
Ran into the same issue, I fixed it in this fork. https://github.com/samze/wiretap you can build the binary here and it should work with cf push